core/keyserver/keyserver.go

```diff
@@ -104,7 +99,7 @@ func (s *Server) getEntry(ctx context.Context, userID, appID string, firstTreeSi
 	})
 	if err != nil {
 		log.Printf("GetLeaves(): %v", err)
-		return nil, grpc.Errorf(codes.Internal, "Failed fetching map leaf")
+		return nil, grpc.Errorf(codes.Internal, "Failed fetching map leaf: %v", err)
```

Comment:
yes, you're right. Ideally, we'd have a grpc interceptor that could strip out the error messages that would have a debug flag we could turn on and off. but this will do for now.